res/Sectors/M1105/Spica.xml

```diff
@@ -32,7 +32,69 @@
     <Subsector Index="O">Kurfane</Subsector>
     <Subsector Index="P">Syzygy</Subsector>
   </Subsectors>
-
+  <Allegiances>
+    <Allegiance Code="SoCf" Base="So">Solomani Confederation</Allegiance>
```

Comment:
Standard 4-character allegiance codes are globally provided and shouldn't be specified in metadata files.

Non-standard 4-character allegiance codes are not allowed.